pkg/services/serviceaccounts/extsvcaccounts/service.go

```diff
@@ -243,9 +271,12 @@ func (esa *ExtSvcAccountsService) ManageExtSvcAccount(ctx context.Context, cmd *
 
 // saveExtSvcAccount creates or updates the service account associated with an external service
 func (esa *ExtSvcAccountsService) saveExtSvcAccount(ctx context.Context, cmd *saveCmd) (int64, error) {
+	ctx, span := esa.tracer.Start(ctx, "ExtSvcAccountsService.saveExtSvcAccount")
```

Comment:
Question: wouldn't this be included in the trace of the caller (`ManageExtSvcAccount` in this case)? Same goes for `deleteExtSvcAccount` and `getExtSvcAccountToken`?

Reply:
I'm a bit unsure about your question. The idea of having sub traces is to see the amount of time spent in each. The  `ManageExtSvcAccount` trace will include both sub traces.

Reply:
Ah, sorry, I completely misread the code and didn't realise that it all links together like that (I looked at the implementation for `RemoveExternalService` and just assumed that all methods would only call one other). All makes sense now, thanks for explaining!